### PR TITLE
SC-423 add events how-to documentation

### PR DESCRIPTION
**Pull request by tomsik68**
### Your checklist for this pull request

Thank you for your contribution to the `next-dev-docs` repo. 

🚨Please review the [Pull request guidelines](../contributor-guide/contributor-guide.md#pull-request-guidelines) to this repository before submitting this PR.

- [x] Your code builds clean without any errors or warnings
  
  ```bash
  yarn build
  ```

- [x] You are using approved terminology

  Refer to the [style guide](../contributor-guide/style-guide.md) for help.

### Description 

This PR adds documentation for events. There are three updates here as specified by SC-423:

1. Add generic description of events to platform overview
1. Add a how-to guide for viewing administrator actions
1. Add a how-to guide for viewing authentication logs

### CODE REVIEW

**Review by Will5 — COMMENTED** [review]
LGTM.

One comment. We've been working off of the 'next' docs instead of the 'v1' docs. Would you mind moving your content over there? I think the appropriate section would be the How-To Guides. I'm happy to help if you need.

**Comment by tomsik68** [comment]
> Would you mind moving your content over there?

No problem, I can easily move stuff over there. Could you please point me to the correct directory for how-to guides and the correct file for the platform overview?

**Comment by Will5** [comment]
> No problem, I can easily move stuff over there. @Will5 Could you please point me to the correct directory for how-to guides and the correct file for the platform overview?

Sure, 
- The How-to Guide directory is `/docs/how-to/`
- The Platform Overview path is `/docs/foundations/platform-overview.mdx`
- You may also need to update `sidebars.js`

File: docs/how-to/view-admin-actions.mdx [hunk 1]
```diff
@@ -0,0 +1,152 @@
+---
+last_update:
+    date: 08/21/2023
+---
```
Comment (Will5) [inline docs/how-to/view-admin-actions.mdx:4]:
```suggestion
---
```

File: docs/how-to/view-auth-logs.mdx [hunk 2]
```diff
@@ -0,0 +1,202 @@
+---
+last_update:
+    date: 08/21/2023
+---
```
Comment (Will5) [inline docs/how-to/view-auth-logs.mdx:4]:
```suggestion
---
```

File: docs/how-to/view-auth-logs.mdx [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+title: Viewing Authentication Logs
+id: view-admin-actions
```
Comment (Will5) [inline docs/how-to/view-auth-logs.mdx:6]:
```suggestion
id: view-auth-logs
```

**Review by Will5 — COMMENTED** [review]
LGTM. I think we need to update `sidebars.js` (look for the 'How-to Guides'), and we can remove `/docs/how-to/view-event-details`

**Comment by tomsik68** [comment]
Thanks Will! I believe I addressed all the suggestions in my latest commits.

File: docs/how-to/view-admin-actions.mdx [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+
+- CREATE - The resource servers have been created
+- READ - The resource servers have been accessed
+- UPDATE - The resource servers have been updated
+- DELETE - The resource servers have been deleted
+
+### Actor
+
+This event carries an actor which is set to the entity that has performed
+this operation.
```
Comment (Will5) [inline docs/how-to/view-admin-actions.mdx:97]: Just noticed this. It seems to be an extra Actor header

**Review by Will5 — APPROVED** [review]
Approved. I would probably just remove that (presumably) extra `### Actor`, but then I think we can merge

**Pull request merged**